javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,189 @@
+"use strict";
+
+//Creating global scope variables
+let userName = "";
+let todos = []; //declared through let to be able to modify the array through the filter method
+
+//Creating a function
+function getReply(command) {
+  let commandSplitted = command.split(" "); //Splits the command into separate words for easy search through them
+
+  //Name actions
+  //.toLowerCase is used to get rid of case sensitivity
+  if (
+    command.toLowerCase().includes("my name is") ||
+    command.toLowerCase().includes("i am")
+  ) {
+    let name = commandSplitted[commandSplitted.length - 1];
```

Comment:
This is not reassigned, so could also be const

Reply:
Fixed.